src/app/fdctl/run/tiles/quic.c

```diff
@@ -121,10 +131,15 @@ allow_fds( fd_tile_args_t * args,
            ulong            out_fds_sz,
            int *            out_fds ) {
   (void)args;
-  if( FD_UNLIKELY( out_fds_sz < 2 ) ) FD_LOG_ERR(( "out_fds_sz %lu", out_fds_sz ));
-  out_fds[ 0 ] = 2; /* stderr */
-  out_fds[ 1 ] = 3; /* logfile */
-  return 2;
+  if( FD_UNLIKELY( out_fds_sz < 3 ) ) FD_LOG_ERR(( "out_fds_sz %lu", out_fds_sz ));
```

Comment:
See above comment, you can mirror what Philip did in the shred tile PR it's a bit cleaner

Reply:
it's already doing the same thing